### ISSUE TITLE
Incorrect "dependents" number

### ISSUE THREAD

**Issue by RafalFilipek** [yarnpkg/website#630]
Hi!

https://yarnpkg.com/en/package/styled-props 
https://yarnpkg.com/en/package/lodash
https://yarnpkg.com/en/package/jest
https://yarnpkg.com/en/package/fela

All those packages has Dependents = `1.5m`

<img width="364" alt="popular" src="https://user-images.githubusercontent.com/80960/30151243-f0343890-93ae-11e7-8d6e-df23a7c636fc.png">
<img width="378" alt="unpopular" src="https://user-images.githubusercontent.com/80960/30151244-f0363500-93ae-11e7-94c5-8d71efe1512b.png">

I've tried different languages & incognito.

Thank you!

**Comment by Haroenv** [comment]
You're right, I noticed this last week, but didn't actually get around to fixing it. I think npm changed their database view. 

Duplicate of https://github.com/algolia/npm-search/issues/63

Thanks for reporting!

**Comment by mnpenner** [comment]
The number shown in badges over at https://nodei.co/ seems to be much more reasonable.
